src/content/guidelines/color/overview.md

```diff
@@ -0,0 +1,227 @@
+
+
+
+| Theme | Primary Background       | Token                | Hex value    |    |
+| --- | ----------- | ------------------- | -------------------- | ------------------- |
+| White   | Global Background Light  | `$ui-background`        | <color-block showhex="true" size="xs">#ffffff</color-block>  |
+| Gray 10   | Global Background Light | `$ui-background`  | <color-block showhex="true" size="xs">#f3f3f3</color-block>  |
+| Gray 90  | Global Background Dark   | `$ui-background` | <color-block showhex="true" size="xs">#282828</color-block>    |
+| Gray 100   | Global Background Dark  | `$ui-background`        | <color-block showhex="true" size="xs">#171717</color-block>     |
+
+
+
+
+### Tokens
+
+These themes are implemented with color tokens. In order to tokenize, a set of universal color variables must be determined by common roles and usage; tokens are not based singularly on a color value (i.e. unique hex code). The same color value may be assigned to multiple variables in a theme’s palette when the values have distinctly different roles.
+
+A universal variable can also have multiple associated roles when the color is consistently used across those roles. This allows for uniform color application across themes while giving each theme the freedom to express its own individuality at a more detailed level.  
+
+With this system, non-IBM users can also create their own theme using our color tokens.
+
+### Light theme
+
+Gray 10 components are generally paired with White backgrounds and White components with Gray 10 backgrounds. Some components, such as buttons and toggles, are common across both backgrounds. 
+
+<grid-wrapper col_lg="12" flex="true">
+    <do-dont-example correct=true label="Gray 10 Dropdown on white background.">
+        <img src="images/Light_theme_01.png" alt="Gray 10 Dropdown on White background.">    
+    </do-dont-example>
+    <do-dont-example correct=true label="White Dropdown on a Gray 10 background.">
+        <img src="images/Light_theme_revised.png" alt="White Dropdown on a Gray 10 background.">    
+    </do-dont-example>
+</grid-wrapper>
+
+<grid-wrapper col_lg="12" flex="true">
+    <do-dont-example correct=true label="White Overflow Menu on Gray 10 card.">
+        <img src="images/Light_theme_03_new.png" alt="White Overflow Menu on Gray 10 card.">    
+    </do-dont-example>
+    <do-dont-example label="Avoid use of midtones.">
+        <img src="images/Light_theme_04.png" alt="Avoid use of midtones.">    
+    </do-dont-example>
+</grid-wrapper>
+
+
+![Light theme sample](images/Light_theme_sample_revised.gif)
+
+
+_Components with common colors across both light UI backgrounds_
+
+
+### Dark theme
+
+Gray 90 components are generally paired with Gray 100 backgrounds and Gray 80 components with Gray 90 backgrounds. Some components, such as buttons and toggles, are common across all three dark UI background.
```

Comment:
Gray 90 components are paired with Gray 100 backgrounds **to make the Gray 100 dark theme** and Gray 80 components with Gray 90 backgrounds **to make the Gray 90 dark theme**. Some components, such as buttons and toggles, are **share the same values across both the dark themes**.